csharp-advanced-topics/VolatileInterlockedLockInCsharp/VolatileInterlockedLockInCsharp/Program.cs

```diff
@@ -0,0 +1 @@
+﻿Console.WriteLine("Hello World");
```

Comment:
We should be exercising the code here, showing some examples in the console app.

Reply:
This still needs to be addressed